core/src/main/java/org/infinispan/configuration/parsing/CacheParser.java

```diff
@@ -1162,21 +1171,29 @@ protected void parseScatteredCache(ConfigurationReader reader, ConfigurationBuil
       }
    }
 
-   private ConfigurationBuilder getConfigurationBuilder(ConfigurationBuilderHolder holder, String name, boolean template, String baseConfigurationName) {
+   private ConfigurationBuilder getConfigurationBuilder(ConfigurationReader reader, ConfigurationBuilderHolder holder, String name, boolean template, String baseConfigurationName) {
       if (holder.getNamedConfigurationBuilders().containsKey(name)) {
          throw CONFIG.duplicateCacheName(name);
       }
+      boolean ignoreMissingTemplates = reader.getProperty(IGNORE_MISSING_TEMPLATES) != null;
```

Comment:
Should we check the value as well? Setting the property to false would evaluate to true here.

Reply:
It's an undocumented property used only internally